
recordreplay_planner:
  ros__parameters:
    ego_topic: "vehicle_kinematic_state"
    trajectory_topic: "trajectory"
    bounding_boxes_topic: "bounding_boxes"
    heading_weight: 0.1
    vehicle:
      cg_to_front_m: 1.0
      cg_to_rear_m: 1.0
      front_corner_stiffness: 0.1
      rear_corner_stiffness: 0.1
      mass_kg: 1500.0
      yaw_inertia_kgm2: 12.0
      width_m: 2.0
      front_overhang_m: 0.5
      rear_overhang_m: 0.5
